apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatInfoView.kt

```diff
@@ -535,6 +775,165 @@ fun LocalAliasEditor(
   }
 }
 
+// when contact is a "contact card"
+@Composable
+private fun ConnectButton(openedFromChatView: Boolean, chat: Chat, contact: Contact, close: () -> Unit) {
```

Comment:
I think we don't have a way to open chat info for contact cards now anymore (comparing to initial implementation), so we can remove this button. @diogofcunha correct?